### DISCUSSION TITLE
Turn off switch backlight - no exposed option

### DISCUSSION THREAD

**Discussion by paulobueno** [Koenkk/zigbee2mqtt#10216] [Devices]
Hey guys
I have a Moes switch [ZTS-EU_2gang](https://www.zigbee2mqtt.io/devices/ZTS-EU_2gang.html#moes-zts-eu_2gang) and I would like to turn off its backlight (a blue led), but it only exposes `switch (state), linkquality`
On moes [website](https://www.moeshouse.com/collections/smart-switch/products/zigbee-wall-touch-smart-light-switch-with-neutral-wire-no-neutral-wire-no-capacitor-needed-smart-life-tuya-2-3-way-muilti-control-association-hub-required-2-gang-white-us) it is written that there is an option to turn off the backlight
I was wondering if it would be possible to add this functionality as a new exposed feature so I can turn it off 
Is it possible or is there any other option?

**Comment by michau83** [comment]
hi Paulobueno, i use same product with you and i connect this switch with C2652P usb hub. it's gone through well to HA via zigbee2mqtt. however i dont have Neutral line, and i got problem when i turn both gangs on then I could not control with zigbee2mqtt until i turn 1 of them off by switch. do you got the same issue

**Reply by paulobueno** [reply]
Hey @michau83 
Were you able to turn the backlight off?
I'm not having the same issue as you, not sure if it is because I'm using it with a neutral line

**Reply by michau83** [reply]
i dont see any option for turn backlight off

**Reply by 1060778506** [reply]
你好我也没有背光暴露
